sormas-ui/src/main/java/de/symeda/sormas/ui/symptoms/SymptomsForm.java

```diff
@@ -488,6 +493,8 @@ protected void addFields() {
 
 		// Initialize lists
 
+		symptomaticFieldId = SYMPTOMATIC;
```

Comment:
This looks like a pretty redundant variable for me; please remove it and explicitely use SYMPTOMATIC instead